etl/src/main/java/org/pih/hivmigration/etl/sql/Migrator.java

```diff
@@ -83,13 +86,17 @@ public void main() {
                 revert(new UserMigrator());
             }
             if (!shouldRevertOnly) {
-                migrate(new UserMigrator(), limit);
+                migrate(new UserMigrator(), -1);
```

Comment:
Unrelated to de-identification, I changed User and StagingData to ignore the limit, since they are quick and you can run into key errors if  users are missing.